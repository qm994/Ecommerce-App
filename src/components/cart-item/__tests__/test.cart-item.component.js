import React from 'react';
import CartItem from '../cart-item.component';
import { shallow } from 'enzyme';

const initialProp = {
    id: 1,
    imageUrl: "https://i.ibb.co/ZYW3VTp/brown-brim.png",
    name: "Brown Brim",
    price: 25
}

describe('expect to match the CartItem', () => {
    it('it can find the CartItem', () => {
        const wrapper = shallow(<CartItem item={initialProp} />);
        expect(wrapper).toMatchSnapshot(); 
    })
});