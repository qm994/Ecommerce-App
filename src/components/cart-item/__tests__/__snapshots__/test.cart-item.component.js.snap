// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to match the CartItem it can find the CartItem 1`] = `ShallowWrapper {}`;
